rwlib/source/loaders/RWBinaryStream.hpp

```diff
@@ -67,7 +67,7 @@ class RWBStream {
      * @brief Returns a new stream for the data inside this one.
      */
     RWBStream getInnerStream() const {
-        return RWBStream(_dataCur, _currChunkSz);
+        return {_dataCur, _currChunkSz};
```

Comment:
This introduces a lot of warnings because of the ptrdiff_t -> size_t conversion. The type of _currChunkSz should be changed to size_t.

Reply:
With static cast?

Reply:
No. The type of the member should be changed.